R/utils.R

```diff
@@ -1403,10 +1402,16 @@ PolyVtx <- function(n, r = 1L, xc = 0L, yc = 0L, t1 = 0) {
   } else if (n < 3L) {
     abort(message = "'n' must be greater than or equal to 3")
   }
-  stopifnot(is_bare_integerish(x = r, n = 1L, finite = TRUE))
-  stopifnot(is_bare_integerish(x = xc, n = 1L, finite = TRUE))
-  stopifnot(is_bare_integerish(x = yc, n = 1L, finite = TRUE))
-  stopifnot(is_bare_numeric(x = t1, n = 1L))
+  stopifnot(
+    "'r' must be a single, finite number" = is_bare_numeric(x = r, n = 1L) &&
+      is.finite(x = r),
+    "'xc' must be a single, finite number" = is_bare_numeric(x = xc, n = 1L) &&
+      is.finite(x = xc),
+    "'yc' must be a single, finite number" = is_bare_numeric(x = yc, n = 1L) &&
+      is.finite(x = yc),
+    "'t1' must be a single, finite number" = is_bare_numeric(x = t1, n = 1L) &&
+      is.finite(x = t1)
```

Comment:
In the previous version `t1` wasn't checked for finiteness - I just want to confirm that this change was intentional 👌

Reply:
Yes, this is intentional. It was not checked previously as `rlang::is_bare_numeric()` does not have a `finite` parameter like `rlang::is_bare_integerish()` (and I didn't really know what I was doing with the argument validation)